src/simple-inverse-dyn.cpp

```diff
@@ -25,6 +25,8 @@
 #include <dynamic-graph/factory.h>
 
```

Comment:
Nothing to do with this PR but...what's the difference between the inverse-dynamics-balance-controller and this new simple-inverse-dyn? They look rather similar.

Reply:
Yes they are similar except that there is no possibility to add motion tasks on the hands or feet. 
I created this file to have a simpler control scheme. 
I use it to understand the huge inverse-dynamics-balance-controller file.
It is used for simple simulations which make the robot moves its CoM (same behaviour as the ex1 and 2 of the jupyter notebooks).